dash/dash.py

```diff
@@ -588,6 +590,25 @@ def __init__(  # pylint: disable=too-many-statements
             )
         self.setup_startup_routes()
 
+    def _setup_hooks(self):
+        # pylint: disable=import-outside-toplevel
+        from ._hooks import HooksManager
+
+        self._hooks = HooksManager
+        self._hooks.register_setuptools()
+
+        for setup in self._hooks.get_hooks("setup"):
```

Comment:
is this where the difference in structure between the things that are stored in different `_ns` entries shows up? this loop is getting one value per list item; the loop below is getting three.